### PR TITLE
Proper capitalization in tree

### LINKED ISSUE #256: ML Exposures show up in UI as 'Ml'
### Describe the bug

The UI shows 'MI' for ML exposures.

### Steps To Reproduce

When adding an exposure of `type: ml`, the UI shows this exposure as 'Ml' <---lowercase L

### Expected behavior

Expected to see `ML` in the sidebar

### Screenshots and log output

image

**The output of `dbt --version`:**
Used the sample project with latest install. Can also reproduce on `0.20.2`
```
installed version: 1.0.4
   latest version: 1.0.4

Up to date!

Plugins:
  - postgres: 1.0.4 - Up to date!
```

### PR DESCRIPTION
**Pull request by jonstacks**
Resolves #256 

### Description

Add a list of static capitalizations
    
Before this, the ML exposure type was showing up in the sidebar with 'Ml', with the l easily being mistaken for a capital 'I' given the font used.

### Checklist
 - [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [x] I have generated docs locally, and this change appears to resolve the stated issue
 - [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt next" section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-## dbt-core 1.0.4 (TBD)
+## dbt-core 1.0.5 (TBD)
+- Fixed capitalization in UI for exposures of `type: ml` ([#256](https://github.com/dbt-labs/dbt-docs/issues/256))
```
Comment (emmyoop) [inline CHANGELOG.md:2]: Tiny ask - Change this heading to `## dbt-core 1.1.0 (TBD)`.

**Review by emmyoop — CHANGES_REQUESTED** [review]
This looks great.  I tested it out and it works as expected.  After the minor fix to teh cahngelog I'll get it merged in and then I'll reflect the changes over in dbt-core.

File: CHANGELOG.md (hunk 1 above)
Comment (jonstacks) [reply CHANGELOG.md]: @emmyoop, thank you for the review. Fixed in f696cf8

**Review by emmyoop — APPROVED** [review]
Great!  Thanks for the contribution!

**Pull request merged**